src/scss/_components.percentage-card.scss

```diff
@@ -0,0 +1,54 @@
+.cmp-percentage-card-grid {
+  @supports (display: grid) {
+    display: grid;
+    grid-gap: 1rem 0;
+
+    @media (min-width: $bp-md) {
+      grid-template-columns: 1fr 1fr 1fr;
+      justify-items: center;
+      grid-gap: 0;
+    }
+  }
+}
+
+.cmp-percentage-card {
+  margin: 2rem auto;
+  text-align: center;
+
+  @supports (display: grid) {
```

Comment:
Grid here, feels a bit overkill. The content is stacking, that is what it should do naturally. a `text-align: center` with appropriate spacing margins should be sufficient..